scripts/run_e2e_tests.py

```diff
@@ -253,19 +281,14 @@ def build_js_files(dev_mode_setting):
     update_dev_mode_in_constants_js(CONSTANT_FILE_PATH, dev_mode_setting)
     if not dev_mode_setting:
         python_utils.PRINT('  Generating files for production mode...')
+        build.main(args=['--prod_env'])
     else:
         # The 'hashes.json' file is used by the `url-interpolation` service.
         if not os.path.isfile(HASHES_FILE_PATH):
             with python_utils.open_file(HASHES_FILE_PATH, 'w') as hash_file:
                 hash_file.write('{}')
-        try:
-            common.run_cmd(
-                [common.NODE_BIN_PATH, WEBPACK_BIN_PATH, '--config',
-                 'webpack.dev.config.ts'])
-        except subprocess.CalledProcessError as error:
-            python_utils.PRINT(error.output)
-            sys.exit(error.returncode)
-    build.main(args=(['--prod_env'] if not dev_mode_setting else []))
+        build.main(args=[])
```

Comment:
These changes are made such that we have a similar process in the e2e script as it's in start.py script i.e, to run build.py before webpack.